pkgs/by-name/gi/git-worktree-switcher/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  installShellFiles,
+  git,
+  jq,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-worktree-switcher";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "mateusauler";
+    repo = "git-worktree-switcher";
+    rev = "refs/tags/${finalAttrs.version}-fork";
+    hash = "sha256-fjY+ieUI7m/4gb68/1a+90aIc9iMyQortVd7oHN4STw=";
+  };
+
+  buildInputs = [
+    jq
+    git
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    installShellFiles
+  ];
+
+  patches = [
+    ./disable-update.patch # Disable update and auto update functionality
+    ./add_list_option.patch # Add `list` option to be more intuitive (`git worktree list`)
```

Comment:
I don't think we should add functionality to a program via a patch in nixpkgs, could you send this upstream?

Reply:
We already started a discussion on this. I agree that it shouldn't be here.